src/Connection.php

```diff
@@ -234,7 +236,12 @@ protected function getDefaultPostProcessor(): QueryProcessor
     }
 
     /**
-     * @inheritDoc OVERRIDDEN for return type change
+     * OVERRIDDEN for return type change
+     *
+     * {@inheritDoc}
```

Comment:
`{@inheritDoc}` is the strict syntax for inlining the parent PHPDoc. It is not required by many IDEs and utilities because of its ambiguous behavior, but to avoid ambiguity, it is used when overriding.
https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlineinheritdoc.pkg.html

Reply:
Please submit this in a separate PR next time.
 
Add's too much noise to an already convoluted diff.